help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,180 @@
+
+
+### Fill in destination details {#destination-details}
+Overview of target file location
+
+  ![Destination Detail](../../assets/catalog/advertising/acxiom/image-destination-details.png)
+
+* **Name (Required)** - The name the destination will be saved under
+* **Description** -  Short explanation of the destination's purpose
+* **Bucket Name (Required)** - Name of the S3 bucket set up on S3
+* **Folder Path (Required)** - If subdirectories in a bucket are used a path must be defined, or '/' to reference the root path.
+* **File Type** - Currently the only file type Acxiom processing will be expecting is CSV
+* **CSV Options**
+  * **Delimiter** - Field delimiter used for the file.  Options include:
+    * **Pipe** (|)
+    * **Semicolon** (;)
+    * **Tab** (\t)
+    * **Colon** (:)
+    * **Comma** (,) (Default)
+  * **Quote Character** - Includes the selected quote value around all columns in the resulting file.  Options include:
+    * **Null Character** (\u0000) (Default)
+    * **Double Quotes** (")
+  * **Escape Character** - When fields include characters that may interfere with the delimiters or quotes such as double quotes or commas, an escape character would need to be provided so they can be properly evaluated during import.  This value indicates what that escape character should be.  Values include:
+    * **Single Quote** (')
+    * **Back Slach** (\) (Default)
+  * **Empty Value** - If the value does not exist on the source this indicates the value that should stand in for that missing value.  Options include:
+    * **Empty String** () (Default)
+    * **Null** (null)
+    * **Empty String in Double Quotes** ("")
+  * **Null Value** - If the value does exist, but the value is a null value this indicate how that should be represented on the file.  Options include:
+    * **Empty String** ()
+    * **Null** (null) (Default)
+    * **Empty String in Double Quotes** ("")
+  * **Compression Format** - Specifies if the resulting file will be compressed or not.  Current option include GZIP, and None
+  * **Include manifest file** -  This Boolean option appears only when the GZIP option is selected.  When selected a file outlining details on the file included in the zip file is provided.
+<br>  ![CSV Options](../../assets/catalog/advertising/acxiom/image-destination-csv-options.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
```

Comment:
Let's please keep the section from the self-service template:

```
## Activate audiences to this destination {#activate}

>[!IMPORTANT]
> 
>* To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}

Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audiences to this destination.
```

Reply:
Added this into the document